### PR TITLE
add dummy data

### PR DESCRIPTION
**Pull request by djwester**
add dummy data

add another smoke test

add instanbul for code coverage

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "eslint-config-brightspace": "^0.4.0",
     "eslint-plugin-html": "^4.0.1",
     "frau-ci": "^1.33.2",
-    "polymer-cli": "^1.5.7"
+    "polymer-cli": "^1.5.7",
+    "web-component-tester-istanbul": "^0.10.0"
```
Comment (djwester) [inline package.json]: adding in istanbul for measuring code coverage

File: wct.conf.json [hunk 2]
```diff
@@ -4,7 +4,18 @@
 		"browsers": [{
 		  "browserName": "chrome"
 		}]
-	  },
+		},
```
Comment (alogan519) [inline wct.conf.json]: indenting looks off

File: test/components/d2l-rubric-criteria-group.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+'use strict';
+
+suite('<d2l-rubric-criteria-group>', function() {
+
+	var element, sandbox;
+
+	suiteSetup(function() {
+		sandbox = sinon.sandbox.create();
+		element = fixture('basic-criteria');
+    });
```
Comment (alogan519) [inline test/components/d2l-rubric-criteria-group.js]: indenting looks off

**Comment by alogan519** [comment]
A lot of the tabbing looks off and the tab size should be 4, take a look at editorconfig for your editor, hopefully there is a plugin.

**Pull request closed without merging**